src/Service.Tests/CosmosTests/QueryFilterTests.cs

```diff
@@ -861,19 +869,16 @@ public async Task TestFilterOnInnerNestedFields()
         public async Task TestFilterWithEntityNameAlias()
         {
             string gqlQuery = @"{
-                stars(first: 1, " + QueryBuilder.FILTER_FIELD_NAME + @" : {tag : {name : {eq : ""test name""}}})
+                planets(first: 1, " + QueryBuilder.FILTER_FIELD_NAME + @" : {stars : {tag: {name : {eq : ""tag1""}}}})
```

Comment:
why does the order of the query/filtered entities need to change? please add comments in code to clarify intent.

Reply:
here order is not changed...now this graphql query represent you can query at container level only i.e. `planet`.